src/pip/_internal/commands/cache.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import
+
+import logging
+import os
+import textwrap
+
+import pip._internal.utils.filesystem as filesystem
+from pip._internal.cli.base_command import Command
+from pip._internal.cli.status_codes import ERROR, SUCCESS
+from pip._internal.exceptions import CommandError, PipError
+from pip._internal.utils.typing import MYPY_CHECK_RUNNING
+
+if MYPY_CHECK_RUNNING:
+    from optparse import Values
+    from typing import Any, List
+
+
+logger = logging.getLogger(__name__)
+
+
+class CacheCommand(Command):
+    """
+        Inspect and manage pip's cache.
```

Comment:
```suggestion
        Inspect and manage pip's wheel cache.
```

To hint at the existence of the http cache, which this command does not inspect?

Reply:
Just made a commit updating all 3 sentences I'd added that used the phrase “pip's cache” to say “pip's wheel cache”. :+1: